kodilla-spring-intro/src/main/java/com/kodilla/spring/intro/shape/Drawer.java

```diff
@@ -0,0 +1,14 @@
+package com.kodilla.spring.intro.shape;
+
+
+public class Drawer {
+        final Shape shape;
```

Comment:
To chyba jest z Kodilli, ale tez powinna byc private

Reply:
zmienione